src/live_restore/live_restore_private.h

```diff
@@ -103,26 +103,26 @@ struct __wt_live_restore_fs {
 };
 
 /*
- * WT_LIVE_RESTORE_WORK_ITEM --
+ * WTI_LIVE_RESTORE_WORK_ITEM --
  *     A single item of work to be worked on by a thread.
  */
-struct __wt_live_restore_work_item {
+struct __wti_live_restore_work_item {
     char *uri;
-    TAILQ_ENTRY(__wt_live_restore_work_item) q; /* List of URIs queued for background migration. */
+    TAILQ_ENTRY(__wti_live_restore_work_item) q; /* List of URIs queued for background migration. */
 };
 
 /*
- * WT_LIVE_RESTORE_SERVER --
+ * WTI_LIVE_RESTORE_SERVER --
  *     The live restore server object that is kept on the connection. Holds a thread group and the
  *     work queue, with some additional info.
  */
-struct __wt_live_restore_server {
+struct __wti_live_restore_server {
     WT_THREAD_GROUP threads;
     wt_shared uint32_t threads_working;
     WT_SPINLOCK queue_lock;
     uint64_t work_items_remaining;
 
-    TAILQ_HEAD(__wt_live_restore_work_queue, __wt_live_restore_work_item) work_queue;
+    TAILQ_HEAD(__wt_live_restore_work_queue, __wti_live_restore_work_item) work_queue;
```

Comment:
should __wt_live_restore_work_queue also be __wti_live_restore_work_queue?

Reply:
Fixed in 883a8834fe7cdfe53028a33baad62d17639b9ae3